File: src/Hurace.Web/src/app/dtos/live-statistic.dto.ts
import { LiveRaceDataDto } from './live-race-data.dto';
import { RunNumber } from '../models';

export interface LiveStatisticDto {
    skierId: number,
    firstName: string,
    lastName: string,
    countryCode: string,
    raceId: number,
    runNumber: RunNumber,
    raceData: LiveRaceDataDto[],
    totalTime: string,
}

File: src/Hurace/Hurace.Api/Dtos/LiveStatisticDto.cs
using System.Collections.Generic;

namespace Hurace.Api.Dtos
{
    public class LiveStatisticDto
    {
        public int SkierId { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string CountryCode { get; set; }

        public int RaceId { get; set; }

        public int RunNumber { get; set; }

        public IEnumerable<LiveRaceDataDto> RaceData { get; set; }

        public string TotalTime { get; set; }
    }
}
